components/atoms/ReportProblem.js

```diff
@@ -9,59 +15,169 @@ export default function ReportProblem() {
     setSubmitted(true);
   };
 
+  const { items } = useContext(LanguageContext);
+  const language = items.language;
+  const t = language === "en" ? en : fr;
+
   return (
-    <DropDown text="Report a problem or mistake on this page">
+    <DropDown text={t.reportAProblemButtonString}>
       {submitted ? (
         <>
-          <div className="pt-2"></div>
-          <div className="pt-4 pb-2.5 pr-5 pl-5 text-center border border-gray-md rounded bg-gray-light-report">
-            <p className="mb-4 text-h3">Thank you for your help!</p>
-            <p className="font-body pl-6 text-left mb-2 w-56">
-              You will not receive a reply. For enquiries, please&nbsp;
-              <a href="https://www.canada.ca/en/contact/index.html">
-                contact us
-              </a>
-              .
-            </p>
-          </div>
+          <h2 className="text-base font-body mb-4">
+            {t.reportAProblemThankYouForYourHelp}
+          </h2>
+          <p className="text-sm font-body mb-4">
+            {t.reportAProblemYouWillNotBeContacted}
+          </p>
+          <a
+            className="block text-xxs font-body hover:text-custom-blue-link focus:text-custom-blue-link focus:underline text-canada-footer-font"
+            href="mailto:experience@servicecanada.gc.ca"
+          >
+            experience@servicecanada.gc.ca
+          </a>
```

Comment:
This text is too small — let's make it the same as the body font we are using in the "you will not receive a reply" paragraph.